### ISSUE TITLE
Can't publish first version without pre-existing git tag

### ISSUE THREAD

**Issue by briancavalier** [northbrookjs/northbrook#89]
**Code to reproduce the issue:**
Try to publish any package that doesn't have a git tag yet.

**Expected behavior:**
Probably some discussion to be had about what is the right thing to do.  Some thoughts:

1. Assume that the version currently set in package.json is the version to publish.  I think I'd be happy with this.
1. Ask the user to provide a version.  This seems ok, too, but I'd prefer 1.
1. Apply northbrook's inferred major, minor, or patch increment to the version currently set in package.json and publish that new version.  This seems far more surprising to me than 1.

**Actual behavior:**
Publishing fails with:

```
Failed to find any tags
error Command failed with exit code 1.
```

**Versions of packages used:**
```
> yarn list northbrook
yarn list v0.19.1
└─ northbrook@4.5.1
```

**Comment by TylorS** [comment]
Thank you very much for the issue! Fixed now in 4.5.2

**Comment by briancavalier** [comment]
Awessome. Thanks @tylors. I'm enjoying using northbrook on some side projects 😄
